L1Trigger/TrackFindingTracklet/src/ChannelAssignment.cc

```diff
@@ -104,7 +104,7 @@ namespace trklet {
   }
 
   // sets channelId of given TTTrackRef, return false if track outside pt range
-  bool ChannelAssignment::channelId(const TTTrackRef& ttTrackRef, int& channelId) {
+  bool ChannelAssignment::channelId(const TTTrackRef& ttTrackRef, int& channelId) const {
```

Comment:
This doesn't set the value of channelId. Is this function just a dummy place holder?